graylog2-web-interface/src/components/common/EntityFilters/FilterConfiguration/DateRangeForm.tsx

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { Formik, Form, Field } from 'formik';
+import moment from 'moment/moment';
+
+import useUserDateTime from 'hooks/useUserDateTime';
+import AbsoluteDateInput from 'views/components/searchbar/date-time-picker/AbsoluteDateInput';
+import { ModalSubmit } from 'components/common';
+import { Checkbox } from 'components/bootstrap';
+import { isValidDate, toUTCFromTz, adjustFormat } from 'util/DateTime';
+
+import type { ValueFilter } from '../types';
+
+const DATE_SEPARATOR = '><';
+
+type FormValues = {
+  from: string | undefined,
+  until: string | undefined,
+}
+
+const Container = styled.div`
+  padding: 3px 10px;
+  max-width: 250px;
+`;
+
+const Info = styled.p(({ theme }) => css`
+  font-size: ${theme.fonts.size.small};
+  margin: 0 0 10px;
+`);
+
+const Sections = styled.div`
+  margin-bottom: 10px;
+`;
+
+const Section = styled.div`
+  :not(:last-child) {
+    margin-bottom: 10px;
+  }
+`;
+
+const SectionHeader = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 3px;
+`;
+
+const StyledLabel = styled.label`
+  margin: 0;
+`;
+
+const StyledCheckbox = styled(Checkbox)`
+  &.checkbox {
+    margin: 0;
+  }
+`;
+
+const DateTimeFormat = styled.code`
+  padding: 0;
+`;
+
+const ErrorMessage = styled.span(({ theme }) => css`
+  color: ${theme.colors.variant.dark.danger};
+  font-size: ${theme.fonts.size.small};
+  font-style: italic;
+  padding: 3px 3px 9px;
+  height: 1.5em;
+`);
+
+const DateConfiguration = ({ name: fieldName, label, allTimeLabel }: {
```

Comment:
`allTimeLabel` is a bit confusing. We use these props for the `Now` checkbox as well. Let's use more general naming. Like a checkbox label or smth...

Reply:
Makes sense! I updated the label.